components/brave_wallet_ui/common/slices/endpoints/nfts.endpoints.ts

```diff
@@ -81,14 +81,15 @@ export const nftsEndpoints = ({
 
           const nftMetadata = await cache.getNftMetadata(arg)
 
-          nftMetadata.imageURL = nftMetadata.imageURL?.startsWith('data:image/')
-            ? nftMetadata.imageURL
-            : (await cache.getIpfsGatewayTranslatedNftUrl(
-                nftMetadata.imageURL || ''
-              )) || undefined
-
           return {
-            data: nftMetadata
+            data: {
+              ...nftMetadata,
+              imageURL: nftMetadata.imageURL?.startsWith('data:image/')
+                ? nftMetadata.imageURL
+                : (await cache.getIpfsGatewayTranslatedNftUrl(
```

Comment:
Let's pull this from simplehash always rather than IPFS. We're moving in that direction anyways so this is easier anyways rather than sometimes loading from gateway.

Reply:
We have an open issue to add the ability to fetch data for a single token from simplehash: https://github.com/brave/brave-browser/issues/32847

Reply:
Will update once the method lands: https://bravesoftware.slack.com/archives/C04JJNFRPK7/p1718722090977159?thread_ts=1718721736.055689&cid=C04JJNFRPK7

Reply:
@kdenhartog  We now use simplehash to get the NFT metadata